Examples/Python/src/Output.cpp

```diff
@@ -601,5 +602,40 @@ void addOutput(Context& ctx) {
     ACTS_PYTHON_MEMBER(nSimulatedEvents);
     ACTS_PYTHON_STRUCT_END();
   }
+
+#define REGISTER_CSV_BFIELD_WRITER_BINDING(suffix, coordinate, grid)      \
```

Comment:
I would probably have gone for a template function instead of the macro, but not critical.

Reply:
Changed to use a template function.

Reply:
the macro is now unused and can be dropped no?

Reply:
Wait why is this still here